packages/injected/src/ariaSnapshot.ts

```diff
@@ -192,7 +193,8 @@ function toAriaNode(element: Element, options?: { forAI?: boolean, refPrefix?: s
     props: {},
     element,
     box: box(element),
-    receivesPointerEvents
+    receivesPointerEvents,
+    active: element.ownerDocument.activeElement === element
```

Comment:
An `iframe` can also be an active element, let's add similar code at line 176 for iframes and have a new test that checks that an iframe is marked as active if it contains currently focused element.